Map_Handler/MainBox.cs

```diff
@@ -66,7 +71,12 @@ private void openMapToolStripMenuItem_Click(object sender, EventArgs e)
 
             if (map_file.ShowDialog() == DialogResult.OK)
             {
-                LoadResourceMaps();
+                if (map_loaded)

```

Comment:
Well resource maps are also utilized by the meta extractor to extract resources that are stored in shared map.
A better implementation would be to create a CloseResourceMaps method and calling it prior to using BlamLibs's tag_extraction methods.